model/registry/deprecated/enduser.yaml

```
@@ -0,0 +1,24 @@
+groups:
+  - id: registry.enduser.deprecated
+    prefix: enduser
+    type: attribute_group
+    brief: Describes deprecated enduser attributes. Complete enduser namespace has been deprecated
+    attributes:
+      - id: id
+        type: string
+        stability: experimental
+        deprecated: use the `user.id` attribute.
```

nit: I think we use this formula everywhere else

```suggestion
        deprecated: Replaced by `user.id` attribute.
```